### DISCUSSION TITLE
Can Modin and Pandas Coexist in a single program ?

### DISCUSSION THREAD

**Discussion by tianlinzx** [modin-project/modin#4627] [Q&A] [answered]
Can Modin and Pandas Coexist in a single program ?
I mean if we replace pandas by  modin.pandas in one file of my program.
Will I have to be "enforced " to use modin.pandas in rest of my program ?

**Comment by mvashishtha** [comment, marked as answer]
@tianlinzx thank you for the question. Yes, modin and pandas can coexist in one program. You can `import modin.pandas as pd` or as whatever name you want (e.g. `import modin.pandas as myname`) in one file, and `import pandas` int the same file or in another file. Mixing modin and pandas **objects**, however, is [not supported](https://github.com/modin-project/modin/issues/3043#issuecomment-1059318253). Does that answer your question?

**Reply by tianlinzx** [reply]
Thanks for you explanation. Much appreciated.
